keycloak/keycloak-admin-ui
# Clients(Authentication): Adds "Export" tab

## Motivation

https://marvelapp.com/prototype/78j1b69/screen/83085244

## Brief Description

Adds "Export" subtab to Authentication settings in Clients

## Verification Steps


1. Go to Clients
2. Create a new OIDC client and make sure "Client authentication" and "Authorization" switches are both enabled
3. Save the client and go to the Authorization -> Export subtab
4. Verify that the "Download" and "Copy" buttons work as expected

## Checklist:
